db/test/test_dataset_eval.py

```diff
@@ -0,0 +1,79 @@
+from db.testing import DatabaseTestCase
+from db import dataset, user, dataset_eval
+import json
+
+
+class DatasetEvalTestCase(DatabaseTestCase):
+
+    def setUp(self):
+        super(DatasetEvalTestCase, self).setUp()
+
+        self.test_user_mb_name = "tester"
+        self.test_user_id = user.create(self.test_user_mb_name)
+        self.test_data = {
+            "name": "Test",
+            "description": "",
+            "classes": [
+                {
+                    "name": "Class #1",
+                    "description": "This is a description of class #1!",
+                    "recordings": [
+                        "0dad432b-16cc-4bf0-8961-fd31d124b01b",
+                        "19e698e7-71df-48a9-930e-d4b1a2026c82",
+                    ]
+                },
+                {
+                    "name": "Class #2",
+                    "description": "",
+                    "recordings": [
+                        "fd528ddb-411c-47bc-a383-1f8a222ed213",
+                        "96888f9e-c268-4db2-bc13-e29f8b317c20",
+                        "ed94c67d-bea8-4741-a3a6-593f20a22eb6",
+                    ]
+                },
+            ],
+            "public": True,
+        }
+        self.test_dataset_id = dataset.create_from_dict(self.test_data, author_id=self.test_user_id)
+
+    def test_create_job(self):
+        job_id = dataset_eval._create_job(self.test_dataset_id)
+        job = dataset_eval.get_job(job_id)
+        self.assertIsNotNone(job)
+        self.assertEqual(job["status"], dataset_eval.STATUS_PENDING)
+
+    def test_get_job(self):
+        job_id = dataset_eval._create_job(self.test_dataset_id)
+        job = dataset_eval.get_job(job_id)
+        self.assertIsNotNone(job)
+        self.assertEqual(type(job), dict)
+
+        self.assertIsNone(dataset_eval.get_job("123e4567-e89b-12d3-a456-426655440000"))
```

Comment:
Did you know that this isn't a valid uuid4?
https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_.28random.29
(Don't worry, you don't need to change it)